src/text/spans.rs

```diff
@@ -79,6 +82,24 @@ impl<'a> Spans<'a> {
             span.reset_style();
         }
     }
+
+    /// Sets the target alignment for this line of text.
+    /// Defaults to: [`None`], meaning the alignment is determined by the rendering widget.
+    ///
+    /// ## Examples
+    ///
+    /// ```rust
+    /// # use std::borrow::Cow;
+    /// # use ratatui::layout::Alignment;
+    /// # use ratatui::text::{Span, Spans};
+    /// # use ratatui::style::{Color, Style, Modifier};
+    /// let mut line = Spans::from("Hi, what's up?").alignment(Alignment::Right);
+    /// assert_eq!(Some(Alignment::Right), line.alignment)
+    /// ```
+    pub fn alignment(self, alignment: Alignment) -> Line<'a> {
+        let line = Line::from(self);
+        line.alignment(alignment)
+    }
```

Comment:
I wanted to introduce this for convenience sake, so users who despite using the deprecated `Spans` wants to use the new feature but can't be bothered to manually switch to `Line`. 

One downside is that it seems out of place. However, once `Spans` is removed for good (if ever?) this would not matter anymore, so I'd let it be for now.

Reply:
Sounds good.